src/web/index.js

```diff
@@ -99,6 +109,8 @@ window.addEventListener('message', ({ source, data: message }) => {
       const hasExtension = selectors.hasExtension(store.getState());
       if (!hasExtension) {
         store.dispatch(actions.ui.setHasExtension({ hasExtension: true }));
+        Metrics.installSuccess();
+        postMessage('setClientUUID', { clientUUID: Metrics.getClientUUID() });
```

Comment:
This does nothing, yet. Still scratching my head on implementing [the `finish-visit` event](https://github.com/mozilla/ThemesRFun/blob/master/docs/metrics.md#finish-visit), which I imagine will require the add-on to make a metric ping and will need the clientUUID